examples/cc3220/README.md

```diff
@@ -0,0 +1,134 @@
+Installing Dependencies & Related Documentation
+===============================================
```

Comment:
I added this document here so Halo could use it until the readme.io page is ready for the public. It keeps me from sending them an email asking them to fetch the latest doc every time I make an update.

I'm happy to remove it if you don't think it's appropriate. Otherwise, I'll remove it when the readme.io page is public or this is released to master.